### PR TITLE
Updates to office-runtime package

### PR DESCRIPTION
**Pull request by mscharlock**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Can give you the Azure DevOps link to our internal repo via email but would prefer not to do on a publicly available thread. I work with the MSFT engineers who are working on office-runtime and custom-functions-runtime and these changes are to support new functionality for custom functions at //build. 
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/office-runtime/office-runtime-tests.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ office-runtime
 Copyright (c) Microsoft Corporation
 */
 
-OfficeRuntime.storage.getItem("foo").then(value => console.log(value));
-OfficeRuntime.storage.getItems(["foo", "bar"]).then(value => console.log(value));
+OfficeRuntime.storage.getItem("foo");
```
Comment (Zlatkovsky) [inline types/office-runtime/office-runtime-tests.ts]: Why remove the `.then`-s?

File: types/office-runtime/index.d.ts [hunk 2]
```diff
@@ -56,101 +56,101 @@ declare namespace OfficeRuntime {
...
-       * Returns a Promise. In the event the Promise does not resolve, returns null. 
+       * Retrieves multiple items from storage based on their key.
+       * Returns a Promise. In the event the Promise does not resolve, returns null.
        * @param keys Keys of items to be removed. Must be an array of strings.
        */
       getItems(keys: string[]): Promise<{ [key: string]: string | null }>;
       /**
-       * Sets multiple items into storage or updates multiple items within IStorage.
-       * Returns a Promise. 
+       * Sets multiple items into storage or updates multiple items within storage.
```
Comment (Zlatkovsky) [inline types/office-runtime/index.d.ts]: Nit: should Storage be capitalized?

**Review by Zlatkovsky — APPROVED** [review]

**Comment by weswigham** [comment]
@mscharlock you done?

**Comment by mscharlock** [comment]
Yep, thanks!

**Pull request merged**